docs/source/create_a_model.mdx

```diff
@@ -0,0 +1,319 @@
+
+For example, [DistilBertForSequenceClassification] is a base DistilBert model with a sequence classification head. The sequence classification head is a linear layer on top of the pooled outputs.
+
+```py
+>>> from transformers import DistilBertForSequenceClassification
+
+>>> model = DistilBertForSequenceClassification.from_pretrained("distilbert-base-uncased")
+===PT-TF-SPLIT===
+>>> from transformers import TFDistilBertForSequenceClassification
+
+>>> tf_model = TFDistilBertForSequenceClassification.from_pretrained("distilbert-base-uncased")
+```
+
+You can easily reuse the checkpoint for another task by switching to another model head. To handle a question answering task, you would load [`DistilBertForQuestionAnswering`]. The question answering head is similar to the sequence classification head, except it is a linear layer on top of the hidden states output.
+
+```py
+>>> from transformers import DistilBertForQuestionAnswering
+
+>>> model = DistilBertForQuestionAnswering.from_pretrained("distilbert-base-uncased")
+===PT-TF-SPLIT===
+>>> from transformers import TFDistilBertForQuestionAnswering
+
+>>> tf_model = TFDistilBertForQuestionAnswering.from_pretrained("distilbert-base-uncased")
+```
+
+## Tokenizer
+
+The last class you need before you use a model for textual data is the [tokenizer](main_classes/tokenizer). A tokenizer prepares the model input by converting raw text to tensors. There are two types of tokenizers you can use with 🤗 Transformers:
+
+- [`PreTrainedTokenizer`]: a Python implementation of a tokenizer.
+- [`PreTrainedTokenizerFast`]: a tokenizer from our Rust-based [🤗 Tokenizer](https://huggingface.co/docs/tokenizers/python/latest/) library. This tokenizer type is significantly faster - especially during batch tokenization - due to it's Rust implementation. The fast tokenizer also offers additional methods like *offset mapping* which maps the original words or characters to their tokens.
+
+Both tokenizers support common methods such as encoding and decoding, adding new tokens, and managing special tokens.
+
+<Tip warning={true}>
+
+Not every model supports a fast tokenizer. Take a look at this [table](index#supported-frameworks) to check if a model has fast tokenizer support.
+
+</Tip>
+
+If you trained your own tokenizer, you can create a tokenizer from your *vocabulary* file:
+
+```py
+>>> from transformers import DistilBertTokenizerFast
+
+>>> fast_tokenizer = DistilBertTokenizerFast.from_pretrained(vocab_file="my_vocab_file.txt")
+```
+
+As with a model, it is generally better and faster to use a pretrained model's vocabulary. Create a [`~transformers.PreTrainedTokenizer.from_pretrained`]:
```

Comment:
That's not completely correct. You have to use a pretrained model's vocabulary if you are using a pretrained model otherwise it will receive inputs that don't make sense.

Also "Create a [`~transformers.PreTrainedTokenizer.from_pretrained`]" will be rendered as "Create a `from_pretrained()` which does not make sense.